[DOC] Add link to access policies (#60352)
grafana/grafana: docs/sources/administration/_index.md

This section includes information for Grafana administrators, team administrators, and users performing administrative tasks:

{{< section >}}

> For information about authorization and authentication for your Grafana Cloud Stack and Grafana Cloud Portal, see the [Grafana Cloud Access Policies documentation](https://grafana.com/docs/grafana-cloud/authentication-and-permissions/access-policies/).

Reviewer: ```suggestion
For information about authorization and authentication for your Grafana Cloud Stack and Grafana Cloud Portal, refer to [Grafana Cloud Access Policies](https://grafana.com/docs/grafana-cloud/authentication-and-permissions/access-policies/).
```
